<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarPage</class>
 <widget class="QWidget" name="CalendarPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>831</height>
   </rect>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>901</width>
     <height>801</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="volunteerTableWidget">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>50</y>
     <width>341</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Apellido</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Conductor</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="notConfirmedVolunteerTable">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>490</y>
     <width>341</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Apellido</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Conductor</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnAddConfirmed">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>400</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubstractConfirmed">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>400</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
